frontend/src/metabase/ui/components/inputs/Switch/Switch.styled.tsx

```diff
@@ -98,7 +98,7 @@ export const getSwitchOverrides = (): MantineThemeOverride["components"] => ({
           height: getSize({ size, sizes: TRACK_HEIGHTS }),
           width: getSize({ size, sizes: TRACK_WIDTHS }),
           cursor: "pointer",
-          [`${getStylesRef("input")}:disabled + &`]: {
+          "input:disabled + &": {
```

Comment:
in sources they used simple selectors, not refs https://github.com/mantinedev/mantine/blob/v6/src/mantine-core/src/Switch/Switch.styles.ts#L109